common/report/class.Report.php

```diff
@@ -175,9 +185,22 @@ public function getSuccesses() {
 	}
 	
 	/**
+	 * returns all info elements
+	 * @return array
+	 */
+	public function getInfos() {
+	    $infos = array();
+	    foreach ($this as $element) {
+	        if ($element->getType() == self::TYPE_INFO) {
+	            $infos[] = $element;
+	        }
+	    }
+	    return $infos;
+	}
+	
+	/**
```

Comment:
wouldn't it be better to have a 
```php
private function getElements($type)
````
And then call it in getSuccesses, getErrors and getInfos ?